bin/test-workflow/6_app_deploy_backend.sh

```diff
@@ -64,6 +64,10 @@ if [[ "$PLATFORM" == "openshift" ]]; then
     --set securityContext.enabled=false \
     --set shmVolume.chmod.enabled=false)
 fi
+# When running KinD locally there may be limited cpu availability.
+if [[ "$CONJUR_PLATFORM" == "kind" ]]; then
+  args+=( --set resources.requests.cpu="0m")
```

Comment:
Is it possible that your local VM might need more CPU allocated to it? I currently have 8 CPUs allocated to my VirtualBox VM. You can use `kubectl` to check the current CPU usage in your node with:
```
kubectl describe nodes
```

The output of this command should show you the available resources for your node:
```
Capacity:
  cpu:                8
  ephemeral-storage:  81527224Ki
  hugepages-2Mi:      0
  memory:             8136496Ki
  pods:               110
```

and then near the bottom, it should show the currently allocated resources on this node:
```
Allocated resources:
  (Total limits may be over 100 percent, i.e., overcommitted.)
  Resource           Requests    Limits
  --------           --------    ------
  cpu                950m (11%)  100m (1%)
  memory             290Mi (3%)  390Mi (4%)
  ephemeral-storage  100Mi (0%)  0 (0%)
  hugepages-2Mi      0 (0%)      0 (0%)
```

Reply:
Thanks, that is it. I haven't rebooted but I can remove this,